src/komodo_utils.cpp

```diff
@@ -1549,17 +1549,21 @@ void komodo_args(char *argv0)
     if ( !chainName.isKMD() )
     {
         BITCOIND_RPCPORT = GetArg("-rpcport", ASSETCHAINS_RPCPORT);
+        IS_KOMODO_TESTNODE = GetArg("-testnode",0);
```

Comment:
You should be able to remove this line entirely. It should already be set above this.